content/tools/schema-parser-options/index.md

```diff
@@ -14,11 +14,14 @@ Use `SchemaParserOptions.newBuilder()` to build an options object to pass to the
 
 Options:
 
+* `contextClass`: If you use a context object to execute your queries, let the parser know about it so that it can add it to data fetchers as an argument.
 * `genericWrappers`: Allows defining your own generic classes that should be unwrapped when matching Java types to GraphQL types.  You must supply the class and the index (zero-indexed) of the wrapped generic type.  For example: If you want to unwrap type argument `T` of `Future<T>`, you must pass `Future.class` and `0`.
 
 * `useDefaultGenericWrappers`: Defaults to `true`.  Tells the parser whether or not to add it's own list of well-known generic wrappers, such as `Future` and `CompletableFuture`.
 * `allowUnimplementedResolvers`: Defaults to `false`.  Allows a schema to be created even if not all GraphQL fields have resolvers.  Intended only for development, it will log a warning to remind you to turn it off for production.  Any unimplemented resolvers will throw errors when queried.
-* `objectMapperConfigurer`: Exposes the Jackson `ObjectMapper` that handles marshalling arguments in method resolvers.  Every method resolver gets its own mapper, and the configurer can configure it differently based on the GraphQL field definition.
+* `missingResolverDataFetcher`: Allows you to provide custom behavior for missing GraphQL fields.
```

Comment:
This feature is not merged yet but I still included it here.
Relevant PR: https://github.com/graphql-java-kickstart/graphql-java-tools/pull/455

Reply:
I merged that one.